internal/pkg/service/buffer/store/export.go

```diff
@@ -178,29 +180,38 @@ func (s *Store) ListExports(ctx context.Context, receiverKey key.ReceiverKey, op
 	_, span := s.tracer.Start(ctx, "keboola.go.buffer.configstore.ListExports")
 	defer telemetry.EndSpan(span, &err)
 
+	// Load sub-objects in parallel, stop at the first error
+	exportsMap := make(map[key.ExportKey]*model.Export)
+	grp, _ := errgroup.WithContext(ctx)
```

Comment:
Najprv este drobny fix, nacitat detaily k exportom mozeme paralelne, pomoze to, ked tam bude tych exportov viac.